### PR TITLE
security advisor changes + tests

### PR DESCRIPTION
**Pull request by orozery**
This PR replaces #1703.
It adds NodeType and A_Name, B_Name fields to the security advisor flow, with extensive unit testing.
Unlike #1703, this PR also includes a functional test.

### CODE REVIEW

File: contrib/packaging/rpm/skydive.te.fedora [hunk 1]
```diff
@@ -50,7 +50,7 @@ sysnet_dns_name_resolve(skydive_t)
 type skydive_etcd_ports_t;
 corenet_port(skydive_etcd_ports_t);
 
-# analyzer DB connect tcp/9200 (ElasticSearch) tcp/2480 (OrientDB)
+# analyzer DB connect tcp/9200 (ElasticSearch) tcp/2480 (OrientDB) tcp/9000 (objectstore)
```
Comment (safchain) [inline contrib/packaging/rpm/skydive.te.fedora]: as the contrib things are not part of the packaging, it should be not be integrated in any case

**Comment by orozery** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by orozery** [comment]
run skydive-k8s-tests

**Comment by orozery** [comment]
@safchain this PR makes changes to the CI to enable functional testing of objectstore.
If you disagree, I can remove the functional testing and we can merge it without it.

@eranra please help merge this PR by discussing with those who are troubled by it.
you can see the functional test as a proof that it is working e2e.

**Comment by safchain** [comment]
@eranra @orozery 

This is really related to the IBM use case. Adding unit and functional tests for this is good to ensure that it will still work for IBM but will consume Skydive CI resources while there are low. Maybe it could be nice to move it somewhere else (other IBM repository) adding a third party CI job to Skydive with an IBM machine ?

**Comment by orozery** [comment]
@safchain I agree.
I removed the CI changes and pushed the modified commit to #1703.
Please merge #1703 and I will close this.

**Comment by hunchback** [comment]
> @eranra @orozery
> 
> This is really related to the IBM use case. Adding unit and functional tests for this is good to ensure that it will still work for IBM but will consume Skydive CI resources while there are low. Maybe it could be nice to move it somewhere else (other IBM repository) adding a third party CI job to Skydive with an IBM machine ?

@orozery  / @safchain / @eranra -  so that CI resources not be needlessly consumed the test can be contributed but disabled by default (so not invoked when running `run tests`) and would require a manual invocation for relevant PRs only. In any case my  recommendation is that it be part of the community code  base.

**Pull request closed without merging**